### PR TITLE
Ignore invalid cert files from cert dir

### LINKED ISSUE #26: When reading ca certs dir, invalid file should be ignored
In https://github.com/nerves-hub/nerves_hub_user_api/blob/main/lib/nerves_hub_user_api/api.ex#L150 , when reading cert files, invalid files should be ignored (README, .git, etc)

### PR DESCRIPTION
**Pull request by jeanparpaillon**
Closes #26

### CODE REVIEW

**Review by ConnorRigby — APPROVED** [review]
i think i agree with this. I also hit a problem related to this earlier this week so its a welcome change. Going to wait for one more approval before merging as i think others may have some opinions

File: lib/nerves_hub_user_api/api.ex
Comment (jeanparpaillon) [inline lib/nerves_hub_user_api/api.ex:157]: I am not sure this is needed...

Comment (jjcarstens) [reply lib/nerves_hub_user_api/api.ex:157]: Agreed. I think we can take this out since order doesn't really matter. Otherwise, LGTM

**Review by fhunleth — APPROVED** [review]

**Pull request merged**
